agent/api_manager/osv_service_api.py

```diff
@@ -156,7 +156,7 @@ def construct_vuln(
                 recommendation=recommendation,
             ),
             technical_detail=f"{vuln.description} \n#### CVEs:\n {', '.join(vuln.cves)}",
-            risk_rating=agent_report_vulnerability_mixin.RiskRating[vuln.risk],
+            risk_rating=agent_report_vulnerability_mixin.RiskRating[vuln.risk.upper()],
```

Comment:
is this always mapping to our risk rating or there are unexpected values?

Reply:
there is `moderate` instead of `medium` and its already handled